### DISCUSSION TITLE
Problem with Seg 11 with Dakota running mpi:

### DISCUSSION THREAD

**Discussion by mlaudu** [snl-dakota/dakota#75] [Q&A] [answered]
I am trying to run an optimization problem using asynch pattern search.
The problem worked well with an older version of Dakota, but I recently
installed version 6.19.0 and I got the following error which caused
mpi to abort:

[compt376:1164726:0:1164726] Caught signal 11 (Segmentation fault: address not mapped to object at address 0x9800681fd8)
==== backtrace (tid:1164726) ====
 0 0x000000000004eb80 killpg()  ???:0
 1 0x000000000000ff80 ddot_()  ???:0
 2 0x0000000000021ae7 HOPSPACK::LapackWrappers::ddot()  ???:0
 3 0x00000000000257cb HOPSPACK::LinConstr::getIneqState()  ???:0
 4 0x00000000000260a2 HOPSPACK::LinConstr::isInequalityFeasible()  ???:0
 5 0x00000000000261bd HOPSPACK::LinConstr::isFeasible()  ???:0
 6 0x000000000006047c HOPSPACK::Hopspack::makeInitialPoint_()  ???:0
 7 0x0000000000062bcf HOPSPACK::Hopspack::setInputParameters()  ???:0
 8 0x0000000000aade65 Dakota::APPSOptimizer::core_run()  ???:0
 9 0x0000000000811926 Dakota::Iterator::run()  ???:0
10 0x00000000004000ab Dakota::IteratorScheduler::run_iterator()  ???:0
11 0x00000000008090e0 Dakota::Environment::execute()  ???:0
12 0x0000000000809800 Dakota::ExecutableEnvironment::execute()  ???:0
13 0x00000000004043be main()  ???:0
14 0x000000000003ad85 __libc_start_main()  ???:0
15 0x000000000040440e _start()  ???:0

Any help would be appreciated.
Thanks.
Musa

**Comment by mlaudu** [comment, marked as answer]
I solved this problem by re-installing Dakota with a different version of Lapack and Blas.

**Reply by jadamstephens** [reply]
On which OS was this?

**Reply by mlaudu** [reply]
It was a High Performance Computing Cluster running a Linux operating system. They upgraded the system and did not install the regular lapack and blas libraries but had openblas and skalapack. I installed the regular lapack and blas libraries but that resulted in the seg 11 error in the blas function ddot_. I reinstalled Dakota using the openblas and skalapack libraries within the cluster and all works well with no errors reported when running appspack optimizer.
